tools/check/check.go

```diff
@@ -404,6 +406,32 @@ func setString(dest *string, _, flag string, defaultValue string) {
 	}
 }
 
+func populateProduct() error {
+	// The SevProduct can come from either product_name or the combination of product and stepping.
+	if *testing.ProductName != "" && (*productString != "" || *stepping != "") {
```

Comment:
Can you add some examples in the comments for ProductName/productString/stepping here? It's a bit unclear what's the difference between productname and productstring. Seems productString and stepping are flags to be parsed in. Is it the expected way to use it?

Reply:
I think that probably --stepping is redundant, but --product without the stepping is needed to not care about the expected machine stepping value. Do the descriptions on the flags not make their use clear enough?